### PR TITLE
Adding Object.values polyfill

### PR DESCRIPTION
**Pull request by ryanrath**
## Description

Internet Explorer ( non-Edge ) does not support `Object.values` so this
polyfill provides that functionality if it does not already exist.

## Motivation and Context
It's nice to have all javascript methods used in an app to work in all supported browsers.

## Tests performed
Manual Test in xdmod docker image
**Test 1:**
- Login to internal dashboard
- Navigate to 'User Management' -> 'Existing Users'
- Click 'Create & Manage Users'
- Select the 'New User' Tab
- Fill out the required information
- Click the 'Save' button
- The user should be successfully created
  - *NOTE: on a docker instance an error will be returned as `sendmail` is not setup. But if you go to / refresh the 'Existing Users' tab then the newly created user will be present*
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: html/gui/lib/ie-object-values-polyfill.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+if (!Object.values) {
+    Object.values = function(obj) {
+        return Object.keys(obj).map(
+          function(key) {
+              return obj[key];
+          }
+        );
+    }
+}
```
Comment (smgallo) [inline html/gui/lib/ie-object-values-polyfill.js]: Need a newline here

File: html/index.php [hunk 2]
```diff
@@ -112,6 +112,7 @@ function isReferrer($referrer)
       <link rel="shortcut icon" href="gui/icons/favicon_static.ico" />
       <script type="text/javascript" src="gui/lib/oldie-console-patch.js"></script>
       <script type="text/javascript" src="gui/lib/oldie-array-includes-polyfill.js"></script>
+      <script type="text/javascript" src="gui/lib/ie-object-values-polyfill.js"></script>
```
Comment (smgallo) [inline html/index.php]: We may want to consider bundling all of these small patches together for performance reasons.

Comment (ryanrath) [reply html/index.php]: That's certainly a possibility, but I'm not sure we gain a lot by bundling just the polyfills / patches. I'm certainly open to seeing some benchmarks though. I would hazard a guess that concatenating / minifying / zipping *all* of our js might lead to some performance gains but then again I'm not sure.

**Comment by ryanrath** [comment]
re: style checks:
I have setup the xdmod-qa repo + lint-diff and have a method for running this pre-commit now.

**Pull request merged**